packages/gafl-webapp-service/src/handlers/multibuy-duplicate-handler.js

```diff
@@ -0,0 +1,7 @@
+export const hasDuplicates = async licencesArray => {
+  const licences = licencesArray.map(licence => `${licence.licenceHolder}${licence.type}${licence.length}${licence.start}`)
+  const duplicates = licences.filter(licenceChecking => {
+    return licences.filter(licenceComparing => licenceChecking === licenceComparing).length > 1
+  })
+  return duplicates.length > 1
+}
```

Comment:
Just one last thing. I think this is more a processor than handler, so please could you switch this into processors (renaming it to `multibuy-processor.js` in the process), thanks